rabbitmq/tests/test.sh

```diff
@@ -18,15 +18,15 @@ if [ "${SKIPBUILD}" -eq 0 ]; then
 
   set -e
   pushd "${PLANDIR}" > /dev/null
-  build
+  DO_CHECK=1 build
```

Comment:
I think DO_CHECK builds and builds for artifacts we test should remain separate.  In most cases we'll probably be fine, but DO_CHECK does modify the behavior of the build process so we won't necessarily be testing the same thing as what builder produces.

Reply:
Removed